src/evm/protocol/uniswap_v4/hooks/hook_handler_creator.rs

```diff
@@ -75,6 +75,13 @@ impl HookHandlerCreator for GenericVMHookHandlerCreator {
             .get("limits_entrypoint")
             .and_then(|bytes| String::from_utf8(bytes.0.to_vec()).ok());
 
+        let is_euler;
+        if let Some(_hooks_data) = params.attributes.get("hook_identifier") {
```

Comment:
Actually Euler will have `euler_v1` as hook_identifier but we can address this in a future PR